lib/MusicBrainz/Server/Edit/Relationship/Delete.pm

```diff
@@ -133,13 +133,14 @@ sub build_display_data
 
     # If no link type exists, we use the fake ID and ensure the next one doesn't clash
     my $link_type_id = $link_type->{id} // $link_type_fake_id++;
+    my $loaded_link_type = $loaded->{LinkType}{$link_type->{id}} if $link_type->{id};
```

Comment:
I had a recent PR adding spaces where the hash key isn't a simple variable or string, so we should probably continue with that.
```suggestion
    my $loaded_link_type = $loaded->{LinkType}{ $link_type->{id} } if $link_type->{id};
```